### DISCUSSION TITLE
Saleor dashboard CORS 405 Method not allowed - local development

### DISCUSSION THREAD

**Discussion by Iseeumhmm** [saleor/saleor#14735] [General] [answered]
I just wanted to post this for someone who may be having a similar issue.  I'm new to Saleor and I've just set it up for the first time.  I was seeing the 405 Method not allowed issue and nothing I could find online was helping.  Everything seemed to be setup correctly between core and dashboard. 

The following solution worked for me by proxying the dashboard in `vite.config.js` as follows:
```
server: {
  port: 9000,
  fs: {
    allow: [searchForWorkspaceRoot(process.cwd()), "../.."],
  },
  proxy: {
    '/api': {
      target: 'http://127.0.0.1:8000/graphql/',
      changeOrigin: true,
      rewrite: (path) => path.replace(/^\/api/, ''),
    }
  }
},
```
And then set `API_URI=/api` in your .env

I believe it is related to the following discussion https://stackoverflow.com/questions/62637876/graphql-error-method-not-allowed-must-be-one-of-options

**Although this does work, please see the correct solution below.**

**Comment by AnthonyDreams** [comment, marked as answer]
#12504

**Reply by Iseeumhmm** [reply]
Thank you for this, I can confirm this works as well.  I searched a ton and didn't find that.

The tldr; is start the backend with `uvicorn saleor.asgi:application --reload` and not `python manage.py runserver`.
